Do-It/Controller/AlertDateTableTableViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AlertDateTableTableViewController.swift
+//  Do-It
+//
+//  Created by Cesar F. Chacon on 3/5/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import UIKit
+import Do_ItCore
+
+protocol AlertDateTableViewControllerDelegate: AnyObject {
+    func alertDateTableViewController(_ alertVC: AlertDateTableViewController,
+                                      didSelectDate alertWhen: DateComponents, alertString: String)
+}
+
+class AlertDateTableViewController: UITableViewController {
+    let alertOptions = ["1 day", "3 days", "1 week", "2 weeks", "1 month"]
+     weak var delegate: AlertDateTableViewControllerDelegate?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return alertOptions.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "alertOption", for: indexPath)
+        cell.textLabel?.text = alertOptions[indexPath.row]
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let option = alertOptions[indexPath.row]
+        var dateComponent = DateComponents()
+        switch option {
```

Comment:
Instead of storing the options as a list of strings and then switching on the string, use a struct inside the controller class that models the data, including both the string and the date components. You can reference Aaron's date interval controller for this pattern.